src/helplist.tooltip.js

```diff
@@ -0,0 +1,20 @@
+export default {
+  bugs: {
+    text:
+      "Required. A list of bugs (e.g. Bugzilla or GitHub) that are relevant to this metric. For example, bugs that track its original implementation or later changes to it.",
+    more_info:
```

Comment:
I think we can just call this `link`, for simplicity

```suggestion
    link:
```